index.html

```diff
@@ -1,18 +1,36 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title><%= title %></title>
-    <meta name="description" content="The path to your next open-source contribution."/>
-    <link rel="icon" href="/favicon.svg" type="image/svg+xml"/>
-    <link rel="alternate icon" href="/favicon.ico" type="image/png" sizes="16x16"/>
-    <link rel="apple-touch-icon" href="/apple-touch-icon.png" sizes="180x180"/>
-    <link rel="mask-icon" href="/favicon.svg" color="#FFFFFF"/>
-    <meta name="msapplication-TileColor" content="#FFFFFF"/>
-    <meta name="theme-color" content="#FFFFFF"/>
+    <meta name="description" content="The path to your next open-source contribution." />
+    <link rel="icon" href="/favicon.svg" type="image/svg+xml" />
+    <link rel="alternate icon" href="/favicon.ico" type="image/png" sizes="16x16" />
+    <link rel="apple-touch-icon" href="/apple-touch-icon.png" sizes="180x180" />
+    <link rel="mask-icon" href="/favicon.svg" color="#FFFFFF" />
+    <meta name="msapplication-TileColor" content="#FFFFFF" />
 
-    <meta property="dcterms:modified" content="<%= date %>"/>
+    <!-- For facebook, linkedin etc -->
+    <meta property="og:title" content="sauced" />
```

Comment:
Can we make this Open Sauced
```suggestion
    <meta property="og:title" content="Open Sauced" />
```

Reply:
yeah, should I create a new pr or what. Can u please guide me?